flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontendParser.java

```diff
@@ -56,6 +56,9 @@
 			"The parallelism with which to run the program. Optional flag to override the default value " +
 			"specified in the configuration.");
 
+	public static final Option JOB_ID_OPTION = new Option("i", "jobid", true, "The JobID of the program to submit." +
+			"When used the program will be run under this JobID. This parameter is a guid.");
```

Comment:
"The JobID of the program to submit." is a bit confusing. I would just say, "An optional UUID to use as the JobID. If a job with that JobID already exists, the job will not be run."

Also prefer "UUID" instead of "guid".

Also, you need to mention that it's optional as I don't think that happens automatically.